Engine/Src/Ancona/Core2D/Systems/CameraSystem.cpp

```diff
@@ -31,7 +32,13 @@ CameraComponent::CameraComponent(
 void CameraComponent::Update(float delta)
 {
     auto effectivePosition = GetEffectiveCenter();
-    _view.setCenter(std::round(effectivePosition.x), std::round(effectivePosition.y));
+    std::ostringstream viewStream;
```

Comment:
Don't need all the streams here, they were used for logging statements that have been removed